tools/kochdocs.nim

```diff
@@ -5,7 +5,7 @@ import os, strutils, osproc, sets
 const
   gaCode* = " --doc.googleAnalytics:UA-48159761-1"
   # --warning[LockLevel]:off pending #13218
-  nimArgs = "--warning[LockLevel]:off --hint[Conf]:off --hint[Path]:off --hint[Processing]:off -d:boot --putenv:nimversion=$#" % system.NimVersion
+  nimArgs = "--warning[LockLevel]:off --warning[Deprecated]:off --warning[UnusedImport]:off --warning[RedefinitionOfLabel]:off --hint[Conf]:off --hint[Path]:off --hint[Processing]:off --hint[XDeclaredButNotUsed]:off -d:boot --putenv:nimversion=$#" % system.NimVersion
```

Comment:
This line seems to be the only necessary line to accomplish what your PR title says.